README.md

```diff
@@ -44,54 +62,126 @@ This goes both ways: when explaining something make sure to present enough conte
 
-All of our code is open source and on GitHub. We organize stories and plan sprints with Pivotal Tracker, and maintain our roadmaps \(currently\) in Google Spreadsheets . Individual coding tasks are tracked as Pivotal cards. We use pull requests to review and merge changes. Every PR is peer-reviewed. Please see here for an overview of our [Teams, Services, and Environments](http://docs.dosomething.org/engineering/overview). We communicate on [Slack](slack.md).
+Sharing the responsibility for our work extends to ticket management as well. If you see that there's an issue that needs to be captured in a ticket, make the ticket. If you *truly* don't have time to do this, ask your team for help. 
 
-### Code review and pull requests
+#### Pivotal Tracker
+The following outlines the stages of a ticket in Pivotal Tracker. The buttons on a ticket are _Start_, _Finish_, _Deliver_, _Accept_ OR _Reject_, and they will transition a ticket within the different stages.
 
-We work in branches, generally off of dev \(or master, depending on the project\). We commit often, to capture our thinking. We package up our work as pull requests, and try to keep PRs reasonably small. \(This also makes it more likely someone will review your PR!\)
+##### ⭐️ Feature OR 🐞 Bug Ticket
+1. **Planned**: Ready for current iteration (also estimated if a Feature ticket).
+2. **Unstarted**: No work has begun to address the ticket.
+3. **Started**: Work has begun to address the ticket. In order to start the ticket, it must be estimated first.
+4. **Finished**: Work to address ticket is completed. A pull request has been submitted and is awaiting review.
+5. **Delivered**: The pull request has been merged and deployed to a QA environment. The developer has verified their work on this environment and added QA / testing notes to the ticket. At this stage, there will be review by Product, Design, and/or Data. Just as we expect developers to keep their ticket statuses up to date, we expect the same from Reviewers (Unstarted, In Review, Pass, or Revise).
+![Review States on a ticket](./assets/reviewStates.png). 
+6. **Rejected**: Reviewer (product manager or tech lead) has reviewed the work to address ticket and rejected it.
+7. **Accepted**: Reviewer (product manager or tech lead) has reviewed the work to address ticket and accepted it.
+
+##### ⚙️ Chore Ticket
+1. **Planned**: Ready for current iteration.
+2. **Unstarted**: No work has begun to address the ticket.
+3. **Started**: Work has begun to address the ticket.
+4. **Accepted**: Work to address ticket is completed, pull request was submitted, reviewed and merged.
+
+### Code Review and Pull Requests
+
+We prefer code that's more readable and easier to understand over code that's concise or obfuscated. We strive to use the most understandable variable names in their respective context, and even favor longer, easier to understand variable names over abbreviated ones.
```

Comment:
In my documentation I wrote "Prefer simple, logic based project structures and conceptual naming conventions over point-in-time references"  The first part is the same but wondering if there is room for including the latter?? 

Example... in our data systems we have a lot of field & table names that refer to code names like 'rogue' and 'phoenix' which unnecessarily confuses downstream stakeholders. I would like to push us more to map conceptual definitions like 'web' & 'campaigns' to make things more straightforward.

Reply:
ooo, I changed to:

> We prefer simple, logic based project structures. We prefer code that's more readable and easier to understand over code that's concise or obfuscated. We strive to use conceptual naming conventions over point-in-time references, and even favor longer, easier to understand variable names over abbreviated ones.